notebooks/tou/hpwh_schedule_basic.qmd

```diff
@@ -0,0 +1,175 @@
+    - Consumer receives feedback on energy cost at each billing cycle (monthly).
+    - Switching to a TOU-adapted schedule incurs a one-time "effort cost" (can be fixed or parameterized).
+    - Decision process is repeated each billing cycle (feedback loop).
+
+# 2. Key Variables and Parameters
+
+| Symbol         | Description                                           | Units          |
+|----------------|------------------------------------------------------|----------------|
+| $S$            | Current HPWH schedule (default or TOU-adapted)       | binary         |
+| $U_{HW}$       | Exogenous hot water usage schedule                   | L/hr           |
+| $\lambda$      | Usage multiplier (building characteristic)           | scalar         |
+| $E$            | Electricity consumption profile (from HPWH operation)| kWh            |
+| $r_{on}, r_{off}$ | TOU electricity rate (on-peak, off-peak)     | $/kWh          |
+| $C_{bill}$     | Monthly electricity bill (for water heating)         | $              |
+| $C_{switch}$   | Consumer switching/"hassle" cost                   | $              |
+| $\Delta C$     | Anticipated bill savings from switching              | $              |
+| $C_{comfort}$  | Comfort penalty for unmet demand                     | $              |
+
+# 3. Model Steps
+
+## Step 1: Calculate Baseline
+
+- **Hot Water Usage:** $U_{HW}$ is fixed by schedule and multiplier.
+- **HPWH Default Schedule:** Operates as needed to meet hot water draws.
+- **Simulate:** Run OCHRE with default HPWH schedule to get $E_{default}$, $C_{bill,default}$.
+
+## Step 2: Define TOU-Adaptive Schedule
+
+- **HPWH TOU Schedule:** Only operates HPWH during off-peak hours.
+  - **Note:** Future implementation will allow the agent to specify a more complicated schedule to optimize between comfort and cost. For now, assume the agent can only choose between default schedule  and TOU-adaptive schedule (only runs during off-peak hours).
+- **Simulate:** Run OCHRE with TOU-adaptive HPWH schedule to get $E_{TOU}$, $C_{bill,TOU}$.
+
+## Step 3: Anticipated Savings
+
+- **Calculate:**
+$$
+\Delta C = C_{bill,default} - C_{bill,TOU}
+$$
+- **Net Benefit:**
+$$
+\text{Net Benefit} = \Delta C - C_{switch} - C_{comfort}
+$$
+
+**Note:** The only difference in monthly bill is due to the difference in HPWH operation between the default and TOU-adaptive schedules since we are assuming all other factors remain constant in the OCHRE simulation. Thus the agent can use the difference in monthly bill to calculate the anticipated savings.
+
+## Step 4: Decision Rule
+
+- **If** $\text{Net Benefit} > 0$: Consumer switches to TOU-adapted schedule.
+- **Else:** Remains on default schedule.
+
+## Step 5: Feedback & Iteration
+
+- At each monthly billing cycle, repeat Steps 1–4 using updated consumption and bills.
+- Consumer can reconsider switching if conditions change (e.g., rate structure, switching cost).
+
```

Comment:
@alexhyunminlee @sherryzuo I think you're missing a final step: at the end of the year, what metrics would be calculated / visualizations plotted to help you evaluate?